docs/source/format.rst

```diff
@@ -267,8 +267,8 @@ flexibility when doing a ``POST`` or ``PUT``. If the value is not a
 string (that is, it is a sequence or structure) it is treated as a
 data structure which is turned into a JSON string. If the value is a
 string that begins with ``<@`` then the rest of the string is treated
-as the name of a file to be loaded from the same directory as the YAML
-file. If the value is an undecorated string, that's the value.
+as a filepath to the loaded. The path is relative to the test directory.
```

Comment:
Do you mean "be loaded" instead of "the loaded"?

It's probably worth noting here that traversing out of the local subtree is not allowed. I'd be happy with something like "The path may not include .." or something with similar meaning.